### PR TITLE
ENH: DICOM plugins layout is moved from browser to module panel

### PR DESCRIPTION
**Pull request by MichaelColonel**
Another solution for an issue #5114

### CODE REVIEW

**Comment by lassoan** [comment]
Thanks a lot, this will be great! I'll review and test this tomorrow.

File: Modules/Scripted/DICOM/Resources/UI/DICOM.ui [hunk 1]
```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>372</width>
+    <width>615</width>
```
Comment (lassoan) [inline Modules/Scripted/DICOM/Resources/UI/DICOM.ui:9]: Keep this value the same. Larger value may increase minimum width of the DICOM module.
```suggestion
    <width>372</width>
```

**Review by lassoan — COMMENTED** [review]
This looks great and works well, thanks a lot.

I've just added a trivial comment inline and the DICOM module documentation (https://github.com/Slicer/Slicer/blob/master/Docs/user_guide/modules/dicom.md) needs a small update, too: 
- Update this text: `Plugin selector section: you can choose which plugins will be allowed to examine the selected series for loading. This section is displayed if the double-arrow button on the left side of loadable items list is pushed.`
- Update screenshots with these new ones that I've uploaded: https://github.com/Slicer/Slicer/releases/download/docs-resources/module_dicom_basic_rev02.png,  https://github.com/Slicer/Slicer/releases/download/docs-resources/module_dicom_advanced_rev02.png

**Comment by MichaelColonel** [comment]
I've updated documentation and UI file.

File: Docs/user_guide/modules/dicom.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ Advanced loading (allows loading DICOM data sets using non-default options):
 - **Examine button:** Runs each of the DICOM Plugins on the currently selected series and offers the result in the Loadable items list table.
```
Comment (lassoan) [inline Docs/user_guide/modules/dicom.md:155]: Please update the "Plugin selector section" above (no double-arrow, etc.)

**Review by lassoan — COMMENTED** [review]
Thank you, it is almost perfect, just need to update the "Plugin selector section" description.

File: Docs/user_guide/modules/dicom.md (hunk 2 above)
Comment (MichaelColonel) [reply Docs/user_guide/modules/dicom.md:155]: Could you check, that description is OK?

Comment (lassoan) [reply Docs/user_guide/modules/dicom.md:155]: Yes, it is perfect, thank you.

**Review by lassoan — DISMISSED** [review]
This all looks good now, thank you!
I'll merge it now.

**Comment by lassoan** [comment]
Squashed and merged your changes. Thank you for your contribution!

**Pull request merged**